### PR TITLE
A sample publication page

### PR DESCRIPTION
**Pull request by Alan32Liu**
To address #1544.
[File preview](https://github.com/google/fuzzbench/blob/publication_page/docs/publications.md).

### CODE REVIEW

File: docs/publications.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+FuzzBench has the honor in helping the following publications by providing either an
+evaluation framework or benchmark programs.
+
+<!---
+[//]: # (Only included publications of top venues.)
+[//]: # (Citations are in Chicago style.)
+[//]: # (Include the following links of each publication, if available:)
+[//]: # (1. Full Paper.)
+[//]: # (2. Our FuzzBench report.)
```
Comment (Alan32Liu) [inline docs/publications.md:16]: Would it be a good idea to include our previous report for each work below?

File: docs/publications.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+has been widely used by many research works to evaluate fuzzers and will
+continue to serve on facilitating fuzzing evaluation for both academia and
+the industry.
+You are more than welcome to reach out to us to evaluate your fuzzers.
+
+FuzzBench has the honor in helping the following publications by providing either an
+evaluation framework or benchmark programs.
+
+<!---
+[//]: # (Only included publications of top venues.)
```
Comment (Alan32Liu) [inline docs/publications.md:12]: Would there be any other good papers to include?

**Comment by Navidem** [comment]
Very nice!

You may include this new paper too: https://arxiv.org/abs/2211.11595

**Comment by Alan32Liu** [comment]
> Very nice!
> 
> You may include this new paper too: https://arxiv.org/abs/2211.11595

Thanks, @Navidem!
I noticed this paper is quite new, is it happen to be under peer review?

**Comment by Navidem** [comment]
> I noticed this paper is quite new, is it happen to be under peer review?

Not sure actually.

**Comment by Alan32Liu** [comment]
> > I noticed this paper is quite new, is it happen to be under peer review?
> 
> Not sure actually.

Thanks!
That paper indeed looks promising, thanks bringing it up!
It also found [many new bugs](https://github.com/ispras/oss-sydr-fuzz/blob/master/TROPHIES.md) and was [compared against multiple fuzzers with `FuzzBench`](https://sydr-fuzz.github.io/fuzzbench/).
I will put it in a new arxiv section and move it to Section 2022/2023 when it is published : )

File: docs/publications.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+[FuzzBench](https://dl.acm.org/doi/pdf/10.1145/3468264.3473932)
+[[BibTeX](https://ieeexplore.ieee.org/abstract/document/9787836)]
```
Comment (lszekeres) [inline docs/publications.md:2]: @Alan32Liu This is pointing to a different paper.

Comment (Alan32Liu) [reply docs/publications.md:2]: Thanks! Fixing now : )

Comment (Alan32Liu) [reply docs/publications.md:2]: Fixed in #1569.

**Pull request merged**